<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RollWindow</class>
 <widget class="QWidget" name="RollWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Class Roll</string>
  </property>
  <widget class="QComboBox" name="ClassSelector">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Choose Class</string>
    </property>
   </item>
  </widget>
  <widget class="QListWidget" name="StudentList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>250</width>
     <height>600</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="Present">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>200</y>
     <width>109</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Present</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="Absent">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>220</y>
     <width>109</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Absent</string>
   </property>
  </widget>
  <widget class="QComboBox" name="AbsentReasonSelector">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>240</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Absent Reason</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Late (unexplained)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Late (explained)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sick</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Family</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Appointment (returning)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Appointment (not returning)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Unknown</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="AddStudent">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Student</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RemoveStudent">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove Student</string>
   </property>
  </widget>
  <widget class="QLabel" name="StudentName">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>40</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Select a student from the list</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="StudentNotesArea">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>200</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveNotes">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>400</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteNotes">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>400</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLabel" name="NotesLabel">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>170</y>
     <width>66</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Notes</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="StudentTimetable">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>0</y>
     <width>250</width>
     <height>650</height>
    </rect>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>337</width>
      <height>634</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QLabel" name="Wed1">
       <property name="text">
        <string>Lesson 1</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="Thu1">
       <property name="text">
        <string>Lesson 1</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="Mon4">
       <property name="text">
        <string>Lesson 4</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="Tue1">
       <property name="text">
        <string>Lesson 1</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="Mon1">
       <property name="text">
        <string>Lesson 1</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="Mon3">
       <property name="text">
        <string>Lesson 3</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="Mon2">
       <property name="text">
        <string>Lesson 2</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="Fri1">
       <property name="text">
        <string>Lesson 1</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="Tue2">
       <property name="text">
        <string>Lesson 2</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="Wed2">
       <property name="text">
        <string>Lesson 2</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="Thu2">
       <property name="text">
        <string>Lesson 2</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="Fri2">
       <property name="text">
        <string>Lesson 2</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="Tue3">
       <property name="text">
        <string>Lesson 3</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="Wed3">
       <property name="text">
        <string>Lesson 3</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="Thu3">
       <property name="text">
        <string>Lesson 3</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="Fri3">
       <property name="text">
        <string>Lesson 3</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="Tue4">
       <property name="text">
        <string>Lesson 4</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="Wed4">
       <property name="text">
        <string>Lesson 4</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="Thu4">
       <property name="text">
        <string>Lesson 4</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QLabel" name="Fri4">
       <property name="text">
        <string>Lesson 4</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="SubmitAll">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>570</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit All and Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SubmitStudent">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>270</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
